src/main/functions/wget.ts

```diff
@@ -68,7 +69,7 @@ export async function wget({ exePath, baseUrl, file, targetDir, onProgress }: Wg
     await rename(join(tempdir, decodeURIComponent(file)), target)
 
     Logger.debug('Removing temp directory')
-    await rm(tempdir, { recursive: true, force: true })
+    await rm(dirname(tempdir), { recursive: true, force: true })
```

Comment:
This deletes the entire downloads folder. Because I found that incomplete downloads stayed about and that isn't ideal. This could become a bit of an issue for multiple downloads inside the same app. Not sure how many of them we have

Reply:
Are we saying this is an existing bug? 

You could just make a tempdir inside the targetdir and delete it on startup

Reply:
Sounds good where is the best space to put such a nuke incomplete downloads?